package.json

```diff
@@ -79,6 +79,7 @@
     "react-dom": "16.2.0",
     "react-fa": "5.0.0",
     "react-rnd": "7.3.1",
+    "shpjs": "^3.4.2",
```

Comment:
Please use fixed version.

Reply:
Nice catch, I hate when that happens via `npm i --save`